include/rocksdb/file_checksum.h

```diff
@@ -97,4 +97,8 @@ class FileChecksumList {
 // Create a new file checksum list.
 extern FileChecksumList* NewFileChecksumList();
 
+// Create a buildin Crc32 based file checksum generatory factory object, which
```

Comment:
s/buildin/built-in, s/generatory/generator

Reply:
@riversand963 Thanks for checking the typos, will change accordingly